mne/gui/_coreg_gui.py

```diff
@@ -80,6 +80,20 @@ class CoregModel(HasPrivateTraits):
     mri = Instance(MRIHeadWithFiducialsModel, ())
     hsp = Instance(InstSource, ())
 
+    # this is needed to trigger a filter update when the head shape source
+    # is changed; or is there another way to trigger _inst_points_changed?
+    inst_points = DelegatesTo('hsp')
+
+    # head shape points relevant to the Model depend on the filter
+    points_filter = Any(desc="Index to select a subset of the head shape "
+                        "points")
+    n_omitted = Property(Int, depends_on=['points_filter'])
+    n_omitted_types = Property(depends_on=['n_omitted'])
+
+    points = Property(depends_on=['inst_points', 'points_filter'],
```

Comment:
why not add `use_eeg_locations` to `depends_on` and apply it in `._get_points()`?